public/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="theme-color" content="#000000">
+    <!--
+      manifest.json provides metadata used when your web app is added to the
+      homescreen on Android. See https://developers.google.com/web/fundamentals/web-app-manifest/
+    -->
+    <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+    <!--
```

Comment:
you could also add this after ``link`` tag ```<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">```

Reply:
What does this `stylesheet` do?

Reply:
I have seen it allows us to use the `materialize css` fonts. I have included it. Thanks for the insight.

Reply:
cool